DoorVisualIndicators/DoorVisualIndicators.lua

```diff
@@ -0,0 +1,102 @@
+ModUtil.RegisterMod("DoorVisualIndicators")
+
+local config = {
+  ShowFountainDoorIndictor = true,
+  ShowMinibossDoorIndicator = true
```

Comment:
Worth having the text and visual cues as separate toggles?

Reply:
Eh I don't see too much value there.  We should just have a consistent visual here for first pass.